examples/aurelia-kitchen-sink/src/stories/button.stories.ts

```diff
@@ -0,0 +1,27 @@
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { action, withActions } from '@storybook/addon-actions';
+import { addComponents, storiesOf } from '@storybook/aurelia';
+import { CoolButton } from '../cool-button/cool-button';
+import 'bootstrap/scss/bootstrap.scss';
+
+storiesOf('Button|Basic', module)
```

Comment:
Could we use CSF instead here?

Reply:
The storiesOf format id well tested, already. I think it's safe to assume if CSF works, the storiesOf method will also work.

Reply:
Moved into custom elements and changed to CSF